examples/scatter-matrix-app.py

```diff
@@ -0,0 +1,42 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/diabetes.csv"
+df = pd.read_csv(filepath)
+# fig = px.scatter_matrix(df)
+# fig.show()
+dimensions_list = df.columns.values
+
+app = Dash(__name__)
+
+app.layout = html.Div([
+    html.H1("Scatterplot Matrix of the Correlates of Diabetes",
+            style={'textAlign': 'center'}),
+    html.Div(
+        [html.H3('Choose your dimensions:',
+                 style={"paddingLight": "60px"}),
```

Comment:
I don't think "paddingLight" is a valid css.  It looks fine without any additional padding, so this could be removed.

Reply:
done!